egs/wsj/s5/steps/nnet3/xconfig_to_configs.py

```diff
@@ -159,6 +159,9 @@ def write_config_files(config_dir, all_layers):
     for layer in all_layers:
         try:
             pairs = layer.get_full_config()
+            if layer.descriptors.has_key('input') and 'input' in layer.descriptors['input']['final-string']:
```

Comment:
can you explain what problem you were trying to solve with this change and the change in basic_layers.py?  I suspect that this is not the right fix.